components/content/LoadingImage/LoadingImage.stories.ts

```diff
@@ -0,0 +1,31 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import './LoadingImage';
+
+export default {
+  component: 'diamond-loading-img',
+};
+
+export const Img: StoryObj = {
+  render: (args) => html`
+    <diamond-loading-img
+      ?block="${args.block}"
+      ?responsive="${args.responsive}"
+      ?radius="${args.radius}"
+    >
+      <img
```

Comment:
A loading image won't have the actual image, that's the point. This needs to work with no image tag

Reply:
Erm okay, do you have any suggestions for what you have in mind? Should I give it width and height props that can be set on the component? I don't quite grasp what the intention is here.

Reply:
Yeah, check out the examples in Mobius and Patterns (listed on the ticket) that are trying to achieve the same thing